package.xml

```diff
@@ -13,6 +13,7 @@
   <depend>moveit_core</depend>
   <depend>moveit_visual_tools</depend>
   <depend>rviz_visual_tools</depend>
+  <depend>warehouse-ros-sqlite</depend>
```

Comment:
I guess, because of this rosdep will pick it up and install it automagically.

Reply:
Oh yes, or you just manually install it in the container

Reply:
So would this mean add this to the `moveit_drake.repos`, or `apt install ros-humble-warehouse-ros-sqlite`. Maybe add that to the Dockerfile?

Reply:
How about now? I fixed the package name: https://github.com/moveit/moveit_drake/pull/7/commits/cab01d1deb93a03f42af3246334ae50143bac1a3